makefile

```diff
@@ -35,8 +35,18 @@ omim.owl: omim.ttl mondo_exactmatch_omim.sssom.owl mondo_exactmatch_omimps.sssom
 		query --update sparql/hgnc_links.ru \
 		convert -f ofn -o $@
 
-mondo_genes.csv: omim.owl
+mondo_genes.robot.tsv: omim.owl
```

Comment:
## Rewrite implementation: `awk` --> `pandas`
- [ ] @joeflack4 refactor

[Nico](https://github.com/monarch-initiative/omim/pull/113#pullrequestreview-2110353316): 
> I am weary of the extreme use of awk, but as long as it is dockerized.. I would advice caution on this and focus on building mondolib

[Trish](https://github.com/monarch-initiative/omim/pull/113#issuecomment-2161617795):
> I am also weary of the extreme use of awk and would prefer to find another option.

Haha, this is funny, because I feel the same way. I thought for some reason you guys would probably prefer a ShellScript solution to `pandas`, but that was also when I thought I only needed to do 2 manipulations, but it turned out to be 4.

After I wrote that, I sent this to my friend who heavily uses `awk` and `sed`, who I've been trying to get to use `pandas`. Not sure if you guys are familiar with this meme, lol:

![8t69kz](https://github.com/monarch-initiative/omim/assets/13045020/c53e04e1-8d55-4290-96b2-24f40af6757e)

It should be an easy rewrite into `pandas`, so I'll do that.

Reply:
Thanks, I appreciate your efforts here, but also think something more readable and more easily portable to a common solution in `mondolib` eventually will be helpful longer term :)

Reply:
- [ ] @twhetzel and/or @matentzn review
Done!

Please take a look at the new Python file and refactored make goal.

I also added column sorting. Forgot to do that before, and it's not entirely unimportant.

I re-ran the goal and the output is the same as what I've attached to the release, the only difference being the sorting. I'll update that file shortly.

RE: `mondolib` refactor: I'm sure there's some kind of ROBOT-template-fu that we could move over there, but I'm not sure yet what that would be. I write a lot of code that looks similar, but the ROBOT templates and the modifications I do to create them vary quite a bit.